src/main/resources/org/openpnp/machine/reference/vision/ReferenceFiducialLocator-DefaultPipeline.xml

```diff
@@ -0,0 +1,20 @@
+<pipeline>
```

Comment:
That's it, yes.  I hadn't quite determined which if either was better, if we should favor the new code with the template match, or how anyone felt about this.  From above, I think I'm going to preserve the code which matches keypoints (or circles for that matter) but clean up the default template match code.

The new CvStage extension class receives template image prior to processing.  The template is matched with result in 'match', then captured image is recalled and blob keypoints saved in 'results'.  Template match checkbox on wizard selects between the two, but not sure I did that exactly right.  One or both can generate locations, closest location to expected location wins, and it tries three times to find one (could be changed to something much more robust).